src/main/java/com/siftscience/VerificationCheckRequest.java

```diff
@@ -0,0 +1,44 @@
+package com.siftscience;
+
+import com.siftscience.model.VerificationCheckFieldSet;
+import okhttp3.OkHttpClient;
+import okhttp3.HttpUrl;
+import okhttp3.Response;
+import okhttp3.Request;
+import okhttp3.Credentials;
+import okhttp3.MediaType;
+import okhttp3.RequestBody;
+
+import java.io.IOException;
+
+
+/*
+* The check call is used for verifying the OTP provided by the end user to Sift.
+* Check out https://sift.com/developers/docs/java/verification-api/check for more information on our request/response structure.
+*/
+public class VerificationCheckRequest extends SiftRequest<VerificationResponse> {
```

Comment:
Maybe this should be something like below:
```suggestion
public class VerificationCheckRequest extends SiftRequest<VerificationCheckResponse> {
```
I know the response structure is almost identical but we do say `checked_at` instead of `sent_at` in the documentation. So the expectation would be to use the `checked_at` field instead of `sent_at`

Reply:
Added both `checked_at` and `sent_at` in the **VerificationCheckRequest** itself. Non-null node will be populated based on response, so that we can have a common response class for all Verification scenarios.

Reply:
@arunmlvtec I have updated the PR to have separate response classes for send, resend & check. Now it should be as suggested format by you.